azure-pipelines.yml

```diff
@@ -4,7 +4,20 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
 
 trigger:
-- master
+  batch: true
```

Comment:
I don't think we should be using git flow for this sort of project. Surely we release on a feature basis?